serverrpc/logs.go

```diff
@@ -0,0 +1,62 @@
+package serverrpc
+
+import (
+	"bytes"
+	"io"
+	"time"
+
+	timetypes "github.com/docker/engine-api/types/time"
+	"github.com/golang/glog"
+	"github.com/hyperhq/hyperd/daemon"
+	"github.com/hyperhq/hyperd/types"
+)
+
+func (s *ServerRPC) ContainerLogs(req *types.ContainerLogsRequest, stream types.PublicAPI_ContainerLogsServer) error {
+	glog.V(3).Infof("ContainerLogs with request %s", req.String())
+
+	var since time.Time
+	if req.Since != "" {
+		s, n, err := timetypes.ParseTimestamps(req.Since, 0)
+		if err != nil {
+			return err
+		}
+		since = time.Unix(s, n)
+	}
+
+	buffer := bytes.NewBuffer([]byte{})
+
+	var stop <-chan bool
```

Comment:
`stop` is nil channel here, you should create one via:
`stop := make(chan bool, 1)`